pom.xml

```diff
@@ -30,10 +30,10 @@ under the License.
 
     <groupId>org.apache.flink</groupId>
     <artifactId>flink-shaded</artifactId>
-    <version>1</version>
+    <version>1.0</version>
```

Comment:
to differentiate between versions that have changed in terms of dependencies (aka "major" versions) and versions that change bugs.